component/constraint/CableConstraint.inl

```diff
@@ -135,37 +140,66 @@ void CableConstraint<DataTypes>::getConstraintResolution(const ConstraintParams*
 
     SOFA_UNUSED(cParam);
 
-    double imposed_value=d_value.getValue()[d_valueIndex.getValue()];
+    double imposedValue=d_value.getValue()[d_valueIndex.getValue()];
 
     if(d_valueType.getValue().getSelectedItem() == "displacement") // displacement
     {
-        if(d_maxDispVariation.isSet())
-        {
-            double displacement = d_displacement.getValue();
-            if(imposed_value > displacement && imposed_value-displacement>d_maxDispVariation.getValue())
-                imposed_value = displacement+d_maxDispVariation.getValue();
-
-            if(imposed_value < displacement && imposed_value-displacement<-d_maxDispVariation.getValue())
-                imposed_value = displacement-d_maxDispVariation.getValue();
-        }
-
+        double minForce=-1e99, maxForce=1e99;
```

Comment:
Probably: https://en.cppreference.com/w/cpp/types/numeric_limits/max instead of hard coded values.